crates/globset/Cargo.toml

```diff
@@ -24,10 +24,12 @@ bstr = { version = "0.2.0", default-features = false, features = ["std"] }
 fnv = "1.0.6"
 log = "0.4.5"
 regex = "1.1.5"
+serde = { version = "1.0.104", optional = true }
```

Comment:
Could you please create a `serde1` feature instead of tying the feature name to the name of a crate? This is useful if e.g., serde support for some reason requires an additional dependency.